README.md

```diff
@@ -1,15 +1,97 @@
-# MAX Image Caption Generator Web App
+# Use a deep learning model to filter images in a web application
```

Comment:
I'm not 100% sold on this new title but I think it's the best we've had so far so I'm ok leaving it until we think of something better.